src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php

```diff
@@ -55,6 +55,26 @@ public static function provideBytesAt(): array
         ];
     }
 
+    /**
+     * @dataProvider provideIndexOf
+     */
+    public function testContainsAny(?int $result, string $string, $needle)
+    {
+        $instance = static::createFromString($string);
+
+        $this->assertSame(null !== $instance->indexOf($needle), $instance->containsAny($needle));
```

Comment:
Does copy pasting the method content here actually test something? Shouldn't we create dedicated providers for this new methods?

Reply:
It's better this way to me.